app/app/Http/Controllers/SignUpController.php

```diff
@@ -15,21 +15,21 @@ public function showForm(Request $request)
     {
 		return view('pages.signup.form');
     }
-	
```

Comment:
Did you do these changes on the latest master?

I guess the only change you intended to make was this:
'confirmmessage' => 'A verification code has been sent to ' . $email . '. Check your email inbox or SPAM folder to confirm.',

I see lots more changes and commits in the pull request, though.

Can you get latest on master and rebase your branch on it, then push up again?  Let me know if you run into trouble.